File: Ohtap E Commerce Domain Models/Development/NetCore/Models Objects/Models Objects/Labels/ILabelData.cs
using Ohtap.ECommerce.Domain.Models.Objects.Datasets;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.Labels
{
    public interface ILabelData :  CoreObject.IObjectID, CoreObject.IObjectCode, CoreObject.IObjectName, CoreObject.IObjectOrder, CoreObject.IObjectDescription
    {
        IDataItemList<IDataItemData> Dataset { get; set; }
    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Labels/ILabelData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\Labels;


use  Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface ILabelData extends CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{

}
